lib/ansible/modules/system/iptables.py

```diff
@@ -559,6 +582,7 @@ def main():
 
 # import module snippets
 from ansible.module_utils.basic import *
+import re
```

Comment:
This should probably be moved to the top, it's just `from ansible.module_utils.basic import *` that is special since it gets replaced with the actual code before the python file is sent over the wire...